### PR TITLE
how to install

### PR DESCRIPTION
**Pull request by aeongdesu**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -1029,6 +1029,7 @@ table tr:hover .table-links {
   margin-top: -1px; }
 
 .nav-pills .nav-link.active, .nav-pills .show > .nav-link {
+  box-shadow: 0 1px 4px #acb5f6;
```
Comment (ilsubyeega) [inline assets/css/style.css:1032]: Since button colors are pink, shadows are blue

**Review by ilsubyeega — CHANGES_REQUESTED** [review]
Sugoidesu is using pink color, not white

**Comment by aeongdesu** [comment]
LOOOOOOOOOOOOOOOOOOOOOOOOL
merge without icon lololol

**Comment by ilsubyeega** [comment]
ima just closing this

**Review by ilsubyeega — COMMENTED** [review]
There is no redirection

**Comment by aeongdesu** [comment]
👌

**Pull request closed without merging**